client/extensions/woocommerce/app/settings/shipping/shipping-zone/index.js

```diff
@@ -141,13 +160,19 @@ Shipping.propTypes = {
 export default connect(
 	( state ) => {
 		const loaded = areShippingZonesFullyLoaded( state );
+		const zone = loaded && getCurrentlyEditingShippingZone( state );
+		const isRestOfTheWorld = zone && 0 === Number( zone.id );
```

Comment:
The `Number()` cast is redundant, the zones returned by the API always have a numeric ID.